src/main/kotlin/uk/gov/justice/digital/hmpps/breachnoticeapi/service/BreachNoticeService.kt

```diff
@@ -247,4 +285,5 @@ class BreachNoticeService(
 
     return pdfBytes
   }
+
```

Comment:
This is failing lint checking I think.
```suggestion
delete this empty space.
```